src/cpp/proxy/domainmap.cpp

```diff
@@ -285,7 +285,7 @@ class DomainMap::Worker : public QObject
 
 		log_info("routes loaded with %d entries", allRules.count());
 
-		QMetaObject::invokeMethod(this, "changed", Qt::QueuedConnection);
+		QMetaObject::invokeMethod(this, "doChanged", Qt::QueuedConnection);
```

Comment:
DomainMap::Worker does not have a `doChanged` slot.

Reply:
thanks for catching this one!